src/sentry/integrations/base.py

```diff
@@ -177,6 +177,10 @@ class IntegrationProviderSlug(StrEnum):
     ],
 }
 
+INTEGRATION_PROVIDER_TO_TYPE = {
+    v: k for k, values in INTEGRATION_TYPE_TO_PROVIDER.items() for v in values
+}
+
```

Comment:
We have some providers that have multiple integration types (namely github), so I think this map will end up incomplete.

https://github.com/getsentry/sentry/blob/361e5e59c87fd60aac2c10d8cde4e193e62f4341/src/sentry/integrations/base.py#L153

Reply:
oof, i wonder if we want to restrict these to the main type? i'm not actually sure what the multiple types are used for here. @ameliahsu

Reply:
for the purposes of a metric i'd just want to stick to one, consistently

Reply:
right now the types are only really being used for messaging integration onboarding so that we can query `OrganizationIntegrationsEndpoint` to check if an org has any messaging integrations installed